docs/sources/configuration/_index.md

```diff
@@ -29,6 +29,13 @@ is especially useful in making sure your config files and flags are being read a
 `-log-config-reverse-order` is the flag we run Loki with in all our environments, the config entries are reversed so
 that the order of configs reads correctly top to bottom when viewed in Grafana's Explore.
 
+## Reload At Runtime
+
+Promtail can reload its configuration at runtime. If the new configuration
+is not well-formed, the changes will not be applied.
+A configuration reload is triggered by sending a `SIGHUP` to the Promtail process or
+sending a HTTP POST request to the `/-/reload` endpoint (when the `--server.reload` flag is enabled).
```

Comment:
Is it really `/-/reload`? Looks like simply `/reload` [in your PR](https://github.com/grafana/loki/pull/7247/files#diff-79fdcfc9aa232bdae4374a6008b56fadb3b5cc858e5d716082697b1fef197bdeR113).

I missed that the flag `server.reload` was not renamed to match the config option `enable_runtime_reload`.
Can you please make it `server.enable-runtime-reload` here and in the flag?

After that I'll merge
Thanks!